### PR TITLE
[WIP] Add custom vue-avatar library & update component + snapshot

### PR DESCRIPTION
**Pull request by pascalwengerter**
Feel free to check out the changes locally. We perhaps should merge my repo to it's master branch and do a proper release before introducing it to `web`

### CODE REVIEW

**Comment by fschade** [comment]
pascalwengerter/vue-avatar#21042021_add-lazy-img-loading needs dist to be part of repo

**Review by fschade — APPROVED** [review]
💯, LGMT, maybe we should backport the vue-avatar later to the web repo if they do not apply the pr

File: src/components/avatars/OcAvatar.vue [hunk 1]
```diff
@@ -1,9 +1,24 @@
 <template>
+  <img
+    v-if="src.length > 0"
```
Comment (kulmann) [inline src/components/avatars/OcAvatar.vue:3]: There is an issue: if the image fails to load we also need to fall back to the avatar component.

**Comment by pascalwengerter** [comment]
Closed as outdated, will open a PR that supersedes this soon

**Pull request closed without merging**